app/src/main/java/org/septa/android/app/database/SEPTADatabase.java

```diff
@@ -14,37 +14,73 @@
     /**
      * Current packaged DB version, update number when packaged DB changes
      */
-    private static final int DATABASE_VERSION = 282;
-    private static final String DATABASE_FILE_NAME = "SEPTA.sqlite";
+    private static final String LATEST_DATABASE_API_URL = "https://s3.amazonaws.com/mobiledb.septa.org/latest/latestDb.json";
+
+    // these are left in case the user does not have the most up to date version of the database
+    // modify databaseVersion and databaseFileName when pushing out a new release with a new DB
+    private static int databaseVersion = 282;
```

Comment:
So shouldn't the DB that is shipped with the App follow the same naming convention as the downloaded one?  This way you can keep everything the same?  We should modify the gradle build to download the latest version of the database along with its version number and put the db version into the app.

Reply:
done